decidim-elections/app/views/decidim/elections/elections/election_log.html.erb

```diff
@@ -0,0 +1,182 @@
+<%= render partial: "decidim/shared/component_announcement" %>
+
+<div class="row section election-log"
+  data-election-id="<%= election.id %>"
+  data-api-endpoint-url="<%= bulletin_board_server %>"
+  data-authority-slug="<%= authority_slug %>"
+  data-authority-public-key="<%= authority_public_key %>">
+  <div class="columns large-8">
+    <h1 class="heading3"><%= t(".title") %></h1>
+    <p><%= t(".description") %></p>
+    <div class="mt-sm">
+      <!-- CREATE ELECTION STEP -->
+      <div class="card" id="create-election-step">
+        <div class="card__content">
+          <div class="card__header">
+            <span class="text-small time"></span>
+            <h5 class="card__title"><%= t(".creation_title") %></h5>
+          </div>
+          <div class="card__text">
+            <span class="no-election-created">
+              <span class="card__text--status text-muted"><%= t(".not_created") %></span>
+              <%= t(".creation_description.not_created") %>
+            </span>
+            <span>
+            <span class="election-created hide">
+              <span class="card__text--status success"><%= t(".complete") %></span>
+              <%= t(".creation_description.complete") %>
+            </span>
+          </div>
+          <div class="card__footer--transparent text-small hide">
+            <strong><%= t(".chained_hash") %></strong>: <span class="chained-hash"></span>
+          </div>
+        </div>
+      </div>
```

Comment:
💬 What do you think about adding a partial view with some arguments?

Reply:
I thought about it but then again, a `cell` would maybe make more sense. I started to use partials, but the steps differ in lots of places. That's why I decided to leave it with this pretty long file. 😄